.github/workflows/test-code.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Setup .NET Core 5.0 SDK
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: '5.0.x'
+          dotnet-version: '5.0.201'
```

Comment:
Needs pull from master.

Reply:
@Shane32 You pushed `5.0.201` back into master.

Reply:
Whoops I thought it would be fixed with the merge

Reply:
Fixed